modules/ROOT/partials/misc/autocorrect-usage-notes.adoc

```diff
@@ -0,0 +1,6 @@
+. When the Spell Checker Pro plugin is used, autocorrection is disabled when a non-English language is selected.
++
+When the {pluginname} is used with the xref:introduction-to-tiny-spellchecker.adoc[Spell Checker Pro] Premium plugin, autocorrection is disabled when a non-English language is selected.
++
+Autocorrection can, however, be re-enabled by selecting the *Tools > Autocorrect* menu item.
```

Comment:
Both `Autocorrect` and `Capitalize` are turned off when changing the language (IIRC). This is just one of those annoying side-effects of us using `autocorrect` as a catch-all for all the corrections that occur, but having two different menu items (autocorrect and capitalize). I'm not sure if we need to clarify this.

Reply:
Looking over this again, it seems that there aren't any references to the fact that `capitalization` is disabled by default with all languages. Also, the fact that this partial is included three times on the autocorrect page should be fixed.

Reply:
```suggestion
Autocorrection and capitalization can, however, be re-enabled by selecting the *Tools > Autocorrect* menu item and the "Tools > Capitalization" menu item respectively.
```

Reply:
(discussion): Removed the usage notes.